query-node/mappings/src/content/channel.ts

```diff
@@ -600,9 +590,13 @@ export async function processChannelPaymentFromMember(
             `queried video based on payee address, EXPECTED: ${channel.id}, ACTUAL: ${video.channel.id}`
```

Comment:
The message here should say that the video's `channelId` does not match with the channel that was queried based on reward (or payee) account.

Alternatively you can provide `channelId` as part of the `where` condition when querying for the video, so that you only search within the correct channel to begin with.